pkgs/applications/science/misc/netlogo/default.nix

```diff
@@ -0,0 +1,43 @@
+{ oraclejdk, stdenv, fetchurl, makeWrapper }:
+
+
+stdenv.mkDerivation rec {
+  name = "NetLogo-${version}";
+  version = "6.0.4";
+
+
+  src = fetchurl {
+    url = "https://ccl.northwestern.edu/netlogo/6.0.4/NetLogo-6.0.4-64.tgz";
+    sha256 = "0dcd9df4dfb218826a74f9df42163fa588908a1dfe58864106936f8dfb76acec";
+  };
+
+
+  buildInputs = [ makeWrapper ];
+
+
+  installPhase = ''
+    mkdir -pv $out/share/netlogo
+    cp -rv app readme.md $out/share/netlogo
+
+    # launcher with `cd` is required b/c otherwise the model library isn't usable
+    launcher="$out/bin/netlogo"
+    mkdir -p $out/bin
+    echo "#!/usr/bin/env bash" > $launcher
```

Comment:
`#!/bin/sh` is probably enough and works even in an empty `PATH`

Reply:
Good point, I didn't know that. Thanks!